app/views/batch_invitation_permissions/new.html.erb

```diff
@@ -16,7 +16,7 @@
         </tr>
       </thead>
       <tbody>
-        <% applications_and_permissions(user_object).each do |(application, permissions)|
```

Comment:
This method is still used in the user permissions views, so I think it needs to stay.

Reply:
Sorry, I forgot that Github doesn't care which commit I left the comment on!

I left that comment on "Remove confusing assignment to permissions". What I meant was that that commit could be fixed-up into "Simplify view code checking for persisted?", in case you hadn't spotted that they were related